Automattic/woocommerce-payments
# Use Reporting API in components

Fixes #8519 

#### Changes proposed in this Pull Request
- Added `data` folder 
- Added controller in wcpay 
- Use the data to populate data in the components

#### Testing instructions
- Use branch `add/5005-reporting-api` on server.
- Enable feature with flag `_wcpay_feature_payment_overview_widget `
- On Payments -> Overview, check that Payment Activity Tile is displayed and shows the correct values for Total Payments Volume, Charges, Refunds, Disputes and Fees.

-------------------


**Post merge**
